common/src/main/webapp/less/style.less

```diff
@@ -39,6 +39,11 @@ a {
   color: var(--site-link-color);
 }
 
+// Override SUI's default, which prevented 
+.ui.form .fields .field .ui.input input, .ui.form .field .ui.input input {
+  width: 100%;
```

Comment:
SUI's default of `auto` was preventing input fields from shrinking in small screens.